internal/pkg/service/common/etcdop/key_test.go

```diff
@@ -116,8 +116,18 @@ func TestTypedKeyOperations(t *testing.T) {
 	require.NoError(t, err)
 	assert.Nil(t, kv)
 
-	// Get - not found
-	result, err := k.Get(client).Do(ctx).ResultOrErr()
+	// GetOrNil - not found
+	resultPtr, err := k.GetOrNil(client).Do(ctx).ResultOrErr()
+	require.NoError(t, err)
+	assert.Nil(t, resultPtr)
+
+	// GetOrEmpty - not found
+	result, err := k.GetOrEmpty(client).Do(ctx).ResultOrErr()
```

Comment:
One would expect that `GetOrEmpty` never returns error. I understand that there are possibilities to return error, I was just wandering about better naming.

Reply:
There can always be a network error, etc., if you have a better idea, tell me.